src/renderer/reader/components/ReaderSettings.tsx

```diff
@@ -390,63 +390,65 @@ interface ITable {
 
 const Slider = ({ value, option, set }: { value: string, option: ITable, set: (a: Pick<ReaderConfig, "pageMargins" | "wordSpacing" | "letterSpacing" | "paraSpacing" | "lineHeight">) => void }) => {
     const [currentSliderValue, setCurrentSliderValue] = React.useState(option.defaultValue);
+    const [currentIndex, setCurrentIndex] = React.useState(() => optionsValues[option.parameter].findIndex((el) => el === option.defaultValue));
```

Comment:
This would fail at compile time with TypeScript `strictNullChecks` (or `strict`) ... which is not yet activated in this project unfortunately.
I would recommend `optionsValues[option.parameter]` ==> `(optionsValues[option.parameter] || [])` and also `.findIndex((el) => el === option.defaultValue)` ==> `.findIndex((el) => el === option.defaultValue) || 0`